lib/experimental/fonts/font-library/font-library.php

```diff
@@ -22,7 +22,8 @@
 function gutenberg_init_font_library_routes() {
 	// @core-merge: This code will go into Core's `create_initial_post_types()`.
 	$args = array(
-		'public'       => true,
+		'public'       => false,
+		'_builtin'     => true,
```

Comment:
Following examples in Core:

```suggestion
		'_builtin'     => true,  /* internal use only. don't use this when registering your own post type. */
```